packages/neuron-ui/src/components/ScriptTag/index.tsx

```diff
@@ -41,7 +43,7 @@ const ScriptTag = ({
   }
 
   return (
-    <button type="button" className={styles.tag} onClick={onClick}>
+    <button type="button" className={className || styles.tag} onClick={onClick}>
```

Comment:
Why should we cover all of the styles of `styles.tag`?

Reply:
When merging the code, it was found that the style of ScriptTag was rewritten in Nervos DAO. So I moved the className into Mutilsig Address and passed the prop to ScriptTag.

Reply:
You have covered all of the styles in packages/neuron-ui/src/components/ApproveMultisigTxDialog/approveMultisigTx.module.scss, I guess we can use ``{`${styles.tag} ${className}`}``

Reply:
<img width="799" alt="image" src="https://github.com/nervosnetwork/neuron/assets/12881040/4fc71601-d756-405c-85df-fbad71c3b81d">
Kirl has set them to the same style.

Reply:
thanks, i will fix it

Reply:
> You have covered all of the styles in packages/neuron-ui/src/components/ApproveMultisigTxDialog/approveMultisigTx.module.scss, I guess we can use `` {`${styles.tag} ${className}`} ``

We can use `clsx` from our `utils`.